libraries/botframework-connector/src/auth/certificateServiceClientCredentialsFactory.ts

```diff
@@ -28,32 +31,42 @@ export class CertificateServiceClientCredentialsFactory extends ServiceClientCre
      * @param certificateThumbprint A hex encoded thumbprint of the certificate.
      * @param certificatePrivateKey A PEM encoded certificate private key.
      * @param tenantId Optional. The oauth token tenant.
-     * @param x5c Optional. Enables application developers to achieve easy certificates roll-over in Azure AD:
      * set this parameter to send the public certificate (BEGIN CERTIFICATE) to Azure AD, so that Azure AD can use it to validate the subject name based on a trusted issuer policy.
      */
-    constructor(
-        appId: string,
-        certificateThumbprint: string,
-        certificatePrivateKey: string,
-        tenantId?: string,
-        x5c?: string
-    ) {
+    constructor(appId: string, certificateThumbprintOrx5c?: string, certificatePrivateKey?: string, tenantId?: string);
+
```

Comment:
If you changed the parameter name, isn't that a breaking change? Also, the param documentation is not updated.

Reply:
No, the user will continue sending the same parameter, the name is not being specified.